index.html

```diff
@@ -129,1517 +70,43 @@
-                                    "Properties": {
-                                        "n": {
-                                        "type": "string",
-                                        "const": "5002"
-                                        },
-                                        "v": {
-                                        "@type": ["iot:TransitionTimeData"],
-                                        "type": "number",
-                                        "minimum": 0.0,
-                                        "maximum": 6553.5
-                                        }
-                                    }
-                                    }
-                                ]
-                                }
-                            }
-                            }
-                            </pre>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-
-    </section>
-        
-    <section id="changes" class="appendix">
-        <h2>Recent Specification Changes</h2>
-        <section>
-            <h3 id="changes-from-note-sept2023">Changes from the Group Note 28 September 2023</h3>
-            <ul>
-                <li>Adjusted Figure 1 to not mention commercial products</li>
-                <li>Added BACnet Binding Template to the list of Protocol Binding Templates in Table 2</li>
-                <li>Removed unnecessary heading hierarchies</li>
-                <li>Added mention that the work continues in the WoT Thing Description document</li>
-            </ul>
-        </section>
-        <section>
-            <h3 id="changes-from-note-jan2020">Changes from the Group Note 30 January 2020</h3>
-            <ul>
-                <li>Separated Binding concepts to separate sections</li>
-                <li>Created individual documents for each binding</li>
-                <li>Clarified the binding mechanism and separated the examples from the mechanism</li>
-                <li>Aligned with the WoT Thing Description 1.1 stable information model</li>
-            </ul>
-        </section>
-        <section>
-            <h3 id="changes-from-note-apr2018">Changes from the Group Note 5 April 2018</h3>
-            <ul>
-                <li>Restructured the document</li>
-                <li>Aligned with the recent TD specification changes</li>
-                <li>Aligned with the WoT Thing Description 1.1 stable information model</li>
-                <li>Added default vocabulary terms</li>
-            </ul>
-        </section>
     </section>
 
-    <section id="sec-acknowledgements" class="appendix">
-        <h2>Acknowledgements</h2>
-    
-        <p>
-            Special thanks to all active participants of the W3C Web of Things Interest Group and Working Group
-            for their technical input and suggestions that led to improvements to this document.
-        </p>
-    
+    <section id="conformance">
```

Comment:
I think we should remove the conformance section, as it looks a bit silly in the finally rendered document. What do you think?